recipe/meta.yaml

```diff
@@ -25,26 +37,31 @@ requirements:
     - libtool  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}
     - vs2017_{{ target_platform }}  # [win]
-    - pkg-config  # [not win]
+    - cmake
+    - ninja
+    - arpack   # [not arm64]
```

Comment:
This should not be necessary. `build` is for build tools like compiler, cmake, etc.

Reply:
Well, yes, I thought so too. I did try to change the dependencies to `host` previously as well, but this raised errors or missing dependencies (as it did now again). In this case, on `linux64` it could not find the `arpack` dependency on runtime. It now can find the dependency, so it suggests that something is incorrectly configured somewhere, but I am not 100% sure where. Do you have any clues?

Reply:
arpack should be solved by https://github.com/conda-forge/igraph-feedstock/pull/25#issuecomment-785753702